README.md

```diff
@@ -0,0 +1,110 @@
+
+```
+pip install cruft
+```
+
+### Step 3: Use cruft to create your Mapping Commons project
+In your virtual environment, create a project using the [mapping-commons-cookiecutter](https://github.com/mapping-commons/mapping-commons-cookiecutter) template.
+
+```
+cruft create https://github.com/mapping-commons/mapping-commons-cookiecutter
+```
+
+This kickstarts an interactive session where you declare the following:
+- `project_name`: Name of the project. [defaults to: my-commons-name]
+- `project_description`: Description of the project [defaults to: 'This is the project description.']
+- `github_or_gitlab`: Choose from [1]: github, [2]: gitlab
+- `git_org`: Name of the github (or gitlab) org the project belongs to. [defaults to: my-org]
+- `full_name`: Name of the author [defaults to: 'My Name']
+- `email`: Author's email [defaults to: 'my-name@my-org.org']
+- `license`: Choose from [1]: Yes, [2]: No [**TEST OPTION FOR NOW**]
+- `inverse_mappings`: Do you want to create inverse mappings where the *subject_id* and *object_id* are switched? Choose from [1]: No, [2]: Yes
+
+### Step 4: Create a GitHub/GitLab project
+
+#### GitHub
+
+1. Go to [https://github.com/new] and follow the instructions, being sure to
+   NOT add a README or .gitignore file (this cookiecutter template will take
+   care of this for you).
+
+2. In a terminal, change to the folder your generated project is in and initialize git for the project.
+
+   ```bash
+   cd my-commons-name
+   git init
+   git add .
+   git commit -m 'Initial commit' -a
+   ```
+
+3. Follow the instructions to 'add the remote to your local git repository'.
+
+   ```bash
+   git remote add origin https://github.com/my-user-or-organization/my-commons-name.git
+   git branch -M main
+   git push -u origin main
+   ```
+
+#### GitLab
+
+1. Go to [https://gitlab.com/projects/new#blank_project] and follow the instructions, being sure to
+   NOT add a README file (this cookiecutter template will take
+   care of this for you).
+
+2. In a terminal, change to the folder your generated project is in and follow the instructions to 'push an existing folder'.
+
+   ```bash
+   cd my-commons-name
+   git init --initial-branch=main
+   git remote add origin https://gitlab.com/my-user-or-organization/my-commons-name.git
+   git add .
+   git commit -m 'Initial commit'
+   git push -set-upstream origin main
+   ```
+
+# What does this do?
+
+The following files and directories are autogenerated in the project:
+
+## TODO
+
+This section is under development. If you have questions about this in the meantime, reach out on https://github.com/mapping-commons/mapping-commons-cookiecutter/issues.
+
+
+# Future updates to the project's boilerplate code
+
+In order to be up-to-date with the template, first check if there is a mismatch between the project's boilerplate code and the template by running:
```

Comment:
Add a note that the python environment needs to be active to run this check?

Reply:
Done. I updated this whole section with instructions on how to run the update commands in either the virtual environment or using the Makefile commands.